src/apps/code-editor/src/app/components/FileDrawer/FileDrawer.js

```diff
@@ -82,9 +85,12 @@ export const FileDrawer = memo(function FileDrawer(props) {
 
   function handleSetOpen() {
     // TODO persist to user settings
-    setOpen(!open);
+    setOpen((open) => !open);
   }
 
+  // Handle keyboard shortcut Drawer
```

Comment:
This comment does not add any additional context which the code does not already clearly express.